google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1beta2/StreamWriter.java

```diff
@@ -932,6 +932,7 @@ public void onComplete() {
     public void onError(Throwable t) {
       LOG.fine("OnError called");
       if (streamWriter.shutdown.get()) {
+        abortInflightRequests(t);
```

Comment:
There are two more issues:
1. Even if you abort the inflight request here, the shutdown can still add more requests through writeAllOutstanding();
2. Besides, it is also possible that another thread is calling append, and will contribute another stuck request at https://screenshot.googleplex.com/7pmsyb2zyKDEaGF